source/NVDAObjects/__init__.py

```diff
@@ -417,6 +417,9 @@ def _get_name(self):
 		"""
 		return ""
 
+	#: Type definition for auto prop '_get_role'
+	role: str

```

Comment:
I don't think this annotation is correct since roles are represented by `ROLE_` constants in `controlTypes` and they are integers.

Reply:
Yes you are right @lukaszgo1, good catch.